src/test/java/com/uid2/admin/vertx/SharingServiceTest.java

```diff
@@ -58,38 +58,25 @@ private void mockSiteExistence(Integer... sites){
     }
 
 
-    @Test
-    void listSiteGet(Vertx vertx, VertxTestContext testContext) {
+    @ParameterizedTest
+    @ValueSource(ints = {5, 4})
+    void listSiteGet(int siteId, Vertx vertx, VertxTestContext testContext) {
         fakeAuth(Role.SHARING_PORTAL);
 
         Map<Integer, AdminKeyset> keysets = new HashMap<Integer, AdminKeyset>() {{
-            put(1, new AdminKeyset(1, 5, "test", Set.of(4,6,7), Instant.now().getEpochSecond(),true, true, new HashSet<>()));
-            put(2, new AdminKeyset(2, 7, "test", Set.of(12), Instant.now().getEpochSecond(),true, true, new HashSet<>()));
-            put(3, new AdminKeyset(3, 4, "test", Set.of(5), Instant.now().getEpochSecond(),true, true, Set.of(ClientType.DSP)));
+            put(5, new AdminKeyset(1, 5, "test", Set.of(4,6,7), Instant.now().getEpochSecond(),true, true, new HashSet<>()));
```

Comment:
I'm not sure about this change. It seems the key for the map should be the keyset ID, but we've changed it here to be the site ID.

Reply:
I think you are right, I misunderstand the meaning of this map. I thought I was just to get data, but as mocking the data that return from adminKeysetSnapshot, I think it make sense to use keysetId as key rather than siteid